guides-and-standards/standards--source-control.md

```diff
@@ -0,0 +1,153 @@
+- Either way, the rules in this document as it pertains to code review and working in branches apply to outside collaborators as well.
+- If you plan to actively work on a repository with outside collaborators, it’s recommended to add `CONTRIBUTING.MD` and `CODE_OF_CONDUCT.md` files to the root of the repository to set expectations ahead of time. Default templates for these will be available soon.
+
+## Common pitfalls and tricks to overcome them
+
+Git is notoriously difficult to use. It’s easy to get stuck in a quagmire of git branches and have no idea how to work your way out. If this happens to you, we’ve all been there, don’t panic! ChatGPT and/or Google are a good first option to see if you can identify a solution. If not, post in the #software-questions Slack channel to get help from fellow Arcadian. See below for some common pitfalls and some strategies to avoid or overcome them.
+
+- **Adding too many files or the wrong files into a commit.** Git has two strategies to help you avoid adding the wrong files into your repository. First, there is the `.gitignore` file in which you can specify file names or paths (including with regex!) that should always be ignored by Git. It might make sense to add things like `*.fastq.gz` to this file to avoid adding large sequencing files to GitHub. Second, instead of doing something like `git add` , it’s generally good practice to type out the names of the files you want to add explicitly (`git add 01-analyze-seq-data.ipynb`) so you don’t accidentally add something unintended. If you do add something you didn’t mean to, you can always remove it later!
+- **Starting from the wrong branch.** A branch can start from the `main` branch of a GitHub repo, or any other branch that already exists. Before you start a new branch, check that you’re on the branch you want to start from with `git branch`. If you’re not, use `git checkout <branch name>` to get to the branch you want to start with before creating a new one. It is helpful to customize your terminal of choice to always show the current branch name. See instructions [here](https://askubuntu.com/questions/730754/how-do-i-show-the-git-branch-with-colours-in-bash-prompt).
+- **Merge conflicts.** When two pull requests touch the same line of code in different ways, this usually creates a merge conflict. If you’re working on a project mostly alone, you can avoid merge conflicts by only working on one section of code per branch. If this isn’t possible, or if you’re working with multiple people, have no fear! GitHub can walk you through resolving the merge conflict by having you select the lines of code you wish to keep.
+
+## Tutorials
+
+There are approximately billions of Git and GitHub tutorials online. Here are some that I found interesting:
```

Comment:
should our hello-world repo and the AUG lessons be added here?

Reply:
Added!